packages/repository/src/relations/has-many/has-many.repository.ts

```diff
@@ -6,12 +6,13 @@
 import {Getter} from '@loopback/core';
 import {Filter, Where} from '@loopback/filter';
 import {Count, DataObject, Options} from '../../common-types';
+import {ModelMetadataHelper} from '../../decorators';
 import {Entity} from '../../model';
 import {
   constrainDataObject,
   constrainFilter,
   constrainWhere,
-  EntityCrudRepository,
+  EntityCrudRepository
```

Comment:
Please run `npm run prettier:fix` to fix the style issue. I think the trailing `,` is expected.